mergify_engine/engine/actions_runner.py

```diff
@@ -164,7 +197,9 @@ async def gen_summary(
             summary += STRICT_MODE_DEPRECATION_SASS
         else:
             summary += STRICT_MODE_DEPRECATION_GHES
-
+    summary_Value = _has_merge_action_with_commit_message_mode(pull_request_rules)
+    if summary_Value is not None:
+        summary += summary_Value
```

Comment:
1. the var name should be `summary_value` see https://en.wikipedia.org/wiki/Snake_case
2. the variable name is not really useful here. I'd rather name it `summary_commit_message_deprecation` for example

Reply:
Camel case don't want to let me go :smiling_face_with_tear: